### DISCUSSION TITLE
Can't delete a package with either CLI or UI

### DISCUSSION THREAD

**Discussion by PERES-Richard** [community/community#40407] [Actions]
### Select Topic Area

Bug

### Body

Hello!

Everything is in the title. I just can't delete 2 packages that I pushed with the wrong name. The dropdown button does not display on the UI (see picture) and I got either a 404 for `"knapsack-microservices-aws/knapset-generator"` or a 422 `"Invalid request.\n\nFor 'links/0/schema', {} is not a null."` for `knapsack-microservices-aws%2Fknapset-generator`, using gh cli.

My guess is that there is a '/' in the container image name, which is somehow interpreted by the API. Or maybe I am just missing out something 🤷‍♂️

When I list the package using the cli, I have this name =>  `"knapsack-microservices-aws/knapset-generator"` (see pic) and on the web I have the sanitized URL `knapsack-microservices-aws%2Fknapset-generator`.
It works fine with other package (without '/') so I don't think it's related to an account issue.

Does someone know how to fix this ? I couldn't find anything on the already posted discussions.

Thanks in advance!

![image](https://user-images.githubusercontent.com/43060465/204690856-f885f612-60eb-49ea-829d-50adb8ab404e.png)
![image](https://user-images.githubusercontent.com/43060465/204690497-14c8aa09-bd03-46f5-923d-689600386998.png)
![image](https://user-images.githubusercontent.com/43060465/204691859-730c13d9-c6d3-4447-957a-d7410f51f19e.png)
![image](https://user-images.githubusercontent.com/43060465/204690682-0b420dd1-ae0a-4bb7-a30f-c89dc3abf1d3.png)

**Comment by ossanna16** [comment]
Hi there @PERES-Richard and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
